justfile

```diff
@@ -55,7 +55,7 @@ lint:
 # Mount the tests directory and run a particular command
 @_mount-tests command: (up "postgres s3")
     # The test directory is mounted into the container only during testing
-    docker-compose {{ DOCKER_FILES }} run \
+    docker-compose {{ DOCKER_FILES }} run -it \
```

Comment:
FWIW according to the [docs](https://docs.docker.com/engine/reference/commandline/compose_run/#options), both `-i` and `-t` default to `true` for `docker-compose run` commands.

Reply:
Without this change pdb would not attach even in `test-session` :thinking:

Reply:
That....is very odd 🤔 Ah well, doesn't hurt to have by any means!

Reply:
This is very strange, but for me this change actually _breaks_ the `just test` command. When I remove the `-it` I'm able to run `just test`, although I wasn't trying to attach pdb.